### PR TITLE
global: version bump and travis fix

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

File: requirements-devel.txt [hunk 1]
```diff
@@ -21,12 +21,3 @@
...
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
-#
-# TODO: Add development versions of some important dependencies here to get a
-#       warning when there are breaking upstream changes, e.g.:
-#
-#     -e git+git://github.com/mitsuhiko/werkzeug.git#egg=Werkzeug
-#     -e git+git://github.com/mitsuhiko/jinja2.git#egg=Jinja2
-
--e git+git://github.com/inveniosoftware/invenio-userprofiles.git#egg=invenio-userprofiles
```
Comment (nharraud) [inline requirements-devel.txt:31]: Why do we need to remove the dependency on `invenio-userprofiles`? It is one of the modules which is used by `invenio-accounts-rest`. Same question for invenio-rest.

**Review by nharraud — COMMENTED** [review]
looks good. Just some doubts about requirements-devel.txt changes

File: requirements-devel.txt (hunk 1 above)
Comment (lnielsen) [reply requirements-devel.txt:31]: Mistake from my side :-) Just wanted to remove the TODO comment :-)

Comment (nharraud) [reply requirements-devel.txt:31]: Ok I'll fix it and merge.

**Comment by nharraud** [comment]
LGTM

**Pull request merged**